client/src/modules/vouchers/modals/search.modal.js

```diff
@@ -40,11 +40,11 @@ function VoucherRegistrySearchModalController(ModalInstance, Dates, filters, Not
 
     // convert dates to strings
     if (parameters.dateFrom) {
-      parameters.dateFrom = Dates.util.str(parameters.dateFrom);
+      parameters.dateFrom = moment(parameters.dateFrom).format('YYYY-MM-DD');
```

Comment:
It would be best if you used `bhConstants.date.format` instead of the string `'YYYY-MM-DD'`.  That way, we can easily change the date format if we need to in the future, and we can be guaranteed that all date formats are the same.